conda_env/specs/binstar.py

```diff
@@ -39,6 +39,14 @@ def __init__(self, name=None, **kwargs):
             self.binstar = None
 
     def can_handle(self):
+        result = self._can_handle()
+        if result:
+            print("WARNING: Binstar environments are deprecated and scheduled to be "
```

Comment:
It shows up elsewhere in user messages in this code too.  I actually spent time figuring out how this code path would even be triggered, and I don't think it's in use anywhere.  I don't think there is anything on the internet documented on how you even use it.  So I don't think this should be much of an issue.